### ISSUE TITLE
What python versions are supported/tested?

### ISSUE THREAD

**Issue by jtrip** [bryanveloso/geopatterns#4]
Hello, 
I wanted to check this out but encountered an error that made me believe I might not be using a version of python that is supported but didn't see where any version was specified.

I tried to (sudo) pip install on three different machines:
cloud9 with 2.7 and 3.2
pythonanywhere default bash console
Mac OS X 10.10.2 with sudo and virtualenv

and received the same error:
```  
Could not find a version that satisfies the requirement geopatterns (from versions: )
No matching distribution found for geopatterns
```

Sorry if it is my error!

**Comment by noodlebreak** [comment]
Same here.

**Comment by 01walid** [comment]
I guess there's some kind of packaging problem
@bryanveloso

**Comment by cristobalcl** [comment]
Two years later I had the same problem. I finally installed it this way:

`pip install git+https://github.com/bryanveloso/geopatterns.git#egg=geopatterns`

**Comment by gunpowdergreentea** [comment]
same^^^^^^^^^^^^^^^^^^^^^^

**Comment by bryanveloso** [comment]
Didn't realize people were still grabbing this! It's been a long time since I've worked on this repo, but I can take a look at it this week. Pull requests are also welcome. :)

**Comment by gunpowdergreentea** [comment]
I'm a python newbie and I just realized I can install with `python setup.py install` so I got it working now. Thanks!

**Comment by jtrip** [comment]
Yeah this seems to only be a bug with the pip deployment. Manual installation works fine for me as well, on Python 3. This is quite the blast from the past. 😸

**Comment by Spudsen** [comment]
Not sure how to use the manual installation. Keep on getting a "Could not find README.md" error when trying to manually install :(
